vars/kola.groovy

```diff
@@ -116,15 +116,24 @@ def call(params = [:]) {
             ids += id
             runKola(id, 'run', "--parallel=${parallel} ${args} ${extraArgs}")
         } else {
-            // basic run
-            if (!params['skipBasicScenarios']) {
-                id = marker == "" ? "kola-basic" : "kola-basic-${marker}"
-                ids += id
-                def skipSecureBootArg = ""
+            // Check if the basic tests are registered kola tests or not.
+            def availableTests = shwrapCapture("kola list --json | jq -r '.[].Name'").split()
+            // This essentually checks if the running has https://github.com/coreos/coreos-assembler/pull/3652
+            def basicScenariosBuiltIn = "basic.nvme" in availableTests  
+            id = marker == "" ? "kola-basic" : "kola-basic-${marker}"
+            ids += id
```

Comment:
This should only be done if we actually do a separate basic scenarios kola run (so in the `else` block below).